draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -696,35 +723,9 @@ To this end, Verifiers MUST follow the following steps (or equivalent):
     for the use case at hand. This decision MUST NOT be based on whether
     a Key Binding JWT is provided by the Holder or not. Refer to (#key_binding_security) for
     details.
- 2. Separate the Presentation into the SD-JWT, the Disclosures (if any), and the Key Binding JWT (if provided).
- 3. Validate the SD-JWT:
-    1. Ensure that a signing algorithm was used that was deemed secure for the application. Refer to [@RFC8725], Sections 3.1 and 3.2 for details. The `none` algorithm MUST NOT be accepted.
-    2. Validate the signature over the SD-JWT.
-    3. Validate the Issuer of the SD-JWT and that the signing key belongs to this Issuer.
-    4. Check that the SD-JWT is valid using `nbf`, `iat`, and `exp` claims, if provided in the SD-JWT, and not selectively disclosed.
-    5. Check that the `_sd_alg` claim value is understood and the hash algorithm is deemed secure.
- 4. Process the Disclosures and embedded digests in the SD-JWT as follows:
-    1. For each Disclosure provided:
-       1. Calculate the digest over the base64url-encoded string as described in (#hashing_disclosures).
-    2. (*) Identify all embedded digests in the SD-JWT as follows:
-       1. Find all objects having an `_sd` key that refers to an array of strings.
-       2. Find all array elements that are objects with one key, that key being `...` and referring to a string.
-    3. (**) For each embedded digest found in the previous step:
-       1. Compare the value with the digests calculated previously and find the matching Disclosure. If no such Disclosure can be found, the digest MUST be ignored.
-       2. If the digest was found in an object's `_sd` key:
-          1. If the respective Disclosure is not a JSON-encoded array of three elements, the Verifier MUST reject the Presentation.
-          2. Insert, at the level of the `_sd` key, a new claim using the claim name and claim value from the Disclosure.
-          3. If the claim name already exists at the same level, the Verifier MUST reject the Presentation.
-          4. Recursively process the value using the steps described in (*) and (**).
-       3. If the digest was found in an array element:
-          1. If the respective Disclosure is not a JSON-encoded array of two elements, the Verifier MUST reject the Presentation.
-          2. Replace the array element with the claim value from the Disclosure.
-          3. Recursively process the value using the steps described in (*) and (**).
-    4. If any digests were found more than once in the previous step, the Verifier MUST reject the Presentation.
-    5. Remove all array elements for which the digest was not found in the previous step.
-    6. Remove all `_sd` keys and their contents from the SD-JWT payload.
-    7. Remove the claim `_sd_alg` from the SD-JWT payload.
- 5. If Key Binding is required:
+ 2. Separate the Presentation into the SD-JWT, and the Key Binding JWT (if provided).
+ 3. Process the SD-JWT as defined in (#sd_jwt_verification).
```

Comment:
I think a reference to the steps above instead of these two would be sufficient.

Reply:
sorry, could you please clarify - I think it already references the steps above? instead of which two? I am not sure I understand what change you want...